src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: PrepearedTodo
+};
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => (
+  <div className={classNames('item', { active: !todo.completed })}>
+    <input
+      type="checkbox"
+      checked={todo.completed}
+      readOnly
+    />
+    <h3
+      style={{ display: 'inline-block' }}
+    >
```

Comment:
It's bad practice to use inline styles in such way, use CSS and className attribute instead
```suggestion
    <h3 style={{ display: 'inline-block' }}>
```

Reply:
![image](https://user-images.githubusercontent.com/23210681/151546083-c71a2545-b710-4b7b-8ed1-ebfb29c3d81d.png)